types/2020-08-27/SKUs.d.ts

```diff
@@ -205,12 +205,12 @@ declare module 'stripe' {
         /**
          * Inventory type. Possible values are `finite`, `bucket` (not quantified), and `infinite`.
          */
-        type?: Inventory.Type;
+        type: Inventory.Type;
```

Comment:
This is new as it's now required in the types (the API always required it but it didn't reflect properly)